README.md

```diff
@@ -171,18 +171,19 @@ GoGovSG uses Github Actions and Serverless to deploy to AWS Elastic Beanstalk an
-|:---:|:---:|:---|
-|EB_ENV_(EDU_/HEALTH_)PRODUCTION, EB_ENV_(EDU_/HEALTH_)STAGING|Yes|Elastic Beanstalk environment name|
-|EB_APP_PRODUCTION, EB_APP_STAGING|Yes|Elastic Beanstalk application name|
-|EB_BUCKET_PRODUCTION, EB_BUCKET_STAGING|Yes|S3 bucket used to store the application bundle|
-|PRODUCTION_BRANCH, STAGING_BRANCH|Yes|Name of Git branches for triggerring deployments to production/staging respectively|
-|ECR_URL|Yes|AWS ECR Docker container registry URI to push built images to|
-|ECR_REPO|Yes|Name of repository in AWS ECR containing images|
-|SENTRY_ORG|No|Sentry.io organisation name|
-|SENTRY_PROJECT_PRODUCTION, SENTRY_PROJECT_STAGING|No|Sentry.io project name|
-|SENTRY_URL|No|Sentry.io URL e.g. `https://sentry.io/`|
-|SENTRY_DNS_PRODUCTION,SENTRY_DNS_STAGING|No|Sentry.io endpoint to post client-side errors to|
+|                     Environment Variable                      | Required | Description/Value                                                                   |
+|:-------------------------------------------------------------:|:--------:|:------------------------------------------------------------------------------------|
+| EB_ENV_(EDU_/HEALTH_)PRODUCTION, EB_ENV_(EDU_/HEALTH_)STAGING |   Yes    | Elastic Beanstalk environment name                                                  |
+|               EB_APP_PRODUCTION, EB_APP_STAGING               |   Yes    | Elastic Beanstalk application name                                                  |
+|            EB_BUCKET_PRODUCTION, EB_BUCKET_STAGING            |   Yes    | S3 bucket used to store the application bundle                                      |
+|               PRODUCTION_BRANCH, STAGING_BRANCH               |   Yes    | Name of Git branches for triggerring deployments to production/staging respectively |
+|                            ECR_URL                            |   Yes    | AWS ECR Docker container registry URI to push built images to                       |
+|                           ECR_REPO                            |   Yes    | Name of repository in AWS ECR containing images                                     |
+|                          SENTRY_ORG                           |    No    | Sentry.io organisation name                                                         |
+|       SENTRY_PROJECT_PRODUCTION, SENTRY_PROJECT_STAGING       |    No    | Sentry.io project name                                                              |
+|                          SENTRY_URL                           |    No    | Sentry.io URL e.g. `https://sentry.io/`                                             |
+|           SENTRY_DNS_PRODUCTION,SENTRY_DNS_STAGING            |    No    | Sentry.io endpoint to post client-side errors to                                    |
+|                         API_KEY_SALT                          |   Yes    | Salt used for APIKey hashing, guide to salt generation [here](#salt-generation)     |
```

Comment:
this looks so much cleaner!!

should we shift this line for `API_KEY_SALT` to the "Server" section from lines 100-136 though? it seems that this is more of an internal server env variable, than for deploying to AWS

but then again, the existing env variables under "Server" include e.g. `AWS_S3_BUCKET` and `SENTRY_AUTH_TOKEN` that don't seem to belong there, so it seems like documenting env variables wherever is the norm :')

Reply:
let's clean this up in another PR. 👍